Button.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import * as React from 'react';
+import {
+  Platform,
+  TouchableOpacity,
+  TouchableNativeFeedback,
+} from 'react-native';
+
+type Props = React.ElementConfig<typeof TouchableOpacity> & {
+  children?: React.Node,
+}; // TODO React.ElementProps<typeof TouchableNativeFeedback>
+
+class Button extends React.Component<Props> {
+  renderAndroid() {
+    const { children, ...props } = this.props;
+
+    return (
+      <TouchableNativeFeedback
```

Comment:
`TouchableNativeFeedback` does not have `style` prop. If you passing a styles to this component you should wrap `View` with `TouchableNativeFeedback`

https://facebook.github.io/react-native/docs/touchablenativefeedback